enumcover.go

```diff
@@ -87,11 +88,20 @@ func checkConsts(pass *analysis.Pass, n ast.Node, typeName string) {
 									if lit, ok := value.(*ast.BasicLit); ok {
 										namesForType[unquote(lit.Value)] = true
 									}
+									if lit, ok := value.(*ast.Ident); ok {
+										namesForType[lit.Name] = true
```

Comment:
Hmm, can you give an example of a case where we'd have an ident here?

Reply:
Where we alias MyEnumD as MyEnumC, one of fields for n.Obj.Decl.Values, should be an *ast.Ident that points to MyEnumD.

This line would have been sufficient to solve the case for "Pointer" and "Ptr", but the issue is that n.Obj, are usually nil for those.